hcp/io/read.py

```diff
@@ -265,20 +265,37 @@ def _hcp_pick_info(info, ch_names):
 
 
 def read_trial_info_hcp(subject, data_type, run_index=0, hcp_path=op.curdir):
-    """ read trial info """
+    """ read trial info
+
+    Parameters
+    ----------
+    subject : str
+        The HCP subject.
+    data_type : str
```

Comment:
what are the possible inputs for data_type ? Might be good to have this here so that user doesn't have to look in the code ...